src/builtins/hmr-runtime.js

```diff
@@ -32,6 +32,9 @@ if ((!parent || !parent.isParcelRequire) && typeof WebSocket !== 'undefined') {
           hmrAccept(global.require, asset.id);
         }
       });
+      // Clear the console after HMR
+      // Check if console.clear function is supported, absence of this caused tests to fail
+      console.clear && console.clear();
```

Comment:
Probably better to use a mock function for the tests like i explained in #1074

Reply:
Sure @DeMoorJasper I will use a mock function to fix it.

Reply:
Hi @DeMoorJasper I have mocked the function console.clear() in test